src/Components/Generator/DynamicInputContainer.js

```diff
@@ -118,7 +118,7 @@ export default class DynamicInputContainer extends preact.Component {
                                         <Text id="input-address" />
                                     </option>
                                 </select>
-                                <div className="icon icon-arrow-down" />
+                                <div className="icon icon-arrow-down" style="top: 8px;" />
```

Comment:
I feel like 10 px are a little bit better, since the svg also contains some margin. What do you think @mal-tee?

Reply:
`10px` is fine :D